.github/workflows/release_tag.yml

```diff
@@ -1,153 +1,15 @@
-      - name: Generate token
-        id: generate_token
-        uses: tibdex/github-app-token@v1
-        with:
-          app_id: ${{ secrets.AUTOMATION_CLIENT_ID }}
-          installation_id: ${{ secrets.AUTOMATION_CLIENT_INSTALLATION }}
-          private_key: ${{ secrets.AUTOMATION_CLIENT_SECRET }}
-      - name: Get previous release tag based on type
-        id: prevrelease
-        uses: ibexa/version-logic-action@master
-        with:
-          currentTag: ${{ env.BUILD_TAG }}
-
-      - name: Checkout Generator
-        uses: actions/checkout@v3
-        with:
-          repository: ibexa/changelog-generator-action
-          ref: v2
-
-      - name: Setup Python environment
-        uses: actions/setup-python@v4
-        with:
-          python-version: '3.x'
-
-      - name: Install pygithub & jira
-        run: |
-          pip install pygithub jira github-action-utils
-      - name: Run generator in a loop
-        id: generator
-        env:
-          INPUT_GITHUB_TOKEN: ${{ steps.generate_token.outputs.token }}
-          INPUT_JIRA_TOKEN: ${{ secrets.JIRA_TOKEN }}
-        run: |
-          cat > input.json <<'EOF'
-          ${{ needs.provide_changed_packages.outputs.matrix }}
-          EOF
-          export INPUT_BARE=True
-          echo "${{ github.repository }} ${{ env.BUILD_TAG }} change log" >> generator_output
-          echo "" >> generator_output
-          echo "Changes since ${{ steps.prevrelease.outputs.previousTag }}" >> generator_output
-          echo "" >> generator_output
-          jq -c '.[]' input.json | while read i; do
-            export GITHUB_REPOSITORY=$(jq -r '.name' <<< "$i")
-            export INPUT_PREVIOUSTAG=$(jq -r '.versions[0]' <<< "$i")
-            export INPUT_CURRENTTAG=$(jq -r '.versions[1]' <<< "$i")
-            echo Running against $GITHUB_REPOSITORY
-            echo -n "## " >> generator_output
-            python main.py >> generator_output
-            echo '' >> generator_output
-          done
-          echo "CHANGELOG_OUTPUT<<EOF" >> $GITHUB_ENV
-          echo "$(cat generator_output)" >> $GITHUB_ENV
-          echo "EOF" >> $GITHUB_ENV
-      - name: Create Release
-        id: create_release
-        uses: zendesk/action-create-release@v1
-        with:
-          tag_name: ${{ env.BUILD_TAG }}
-          body: |
-            ${{ env.CHANGELOG_OUTPUT }}
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-
-      - name: Archive markdown
-        uses: actions/upload-artifact@v3
-        with:
-          name: changelogs
-          path: generator_output
+  create_release_for_tag:
+    uses: ibexa/gh-workflows/.github/workflows/release.yml@main
+    secrets:
+      AUTOMATION_CLIENT_ID: ${{ secrets.AUTOMATION_CLIENT_ID }}
+      AUTOMATION_CLIENT_INSTALLATION: ${{ secrets.AUTOMATION_CLIENT_INSTALLATION }}
+      AUTOMATION_CLIENT_SECRET: ${{ secrets.AUTOMATION_CLIENT_SECRET }}
+      JIRA_TOKEN: ${{ secrets.JIRA_TOKEN }}
```

Comment:
I was unable to test `secrets: inherit` in the fork, thus it's safer to explicitly list secrets.

Reply:
Although it might introduce complexity if we ever will have a need to change the invocation of the reusable workflow.

Reply:
> I was unable to test `secrets: inherit` in the fork, thus it's safer to explicitly list secrets.
> Although it might introduce complexity if we ever will have a need to change the invocation of the reusable workflow.

Fine by me, it's already a big improvement :+1: